File: servicio-orquestador/orquestador/src/main/java/co/edu/unicauca/orquestador/fachadaServices/DTORespuesta/RespuestaConsultaPazySalvoDTO.java
package co.edu.unicauca.orquestador.fachadaServices.DTORespuesta;

import java.util.List;

import lombok.Data;

@Data
public class RespuestaConsultaPazySalvoDTO {
    private int codigoEstudiante;
    private List<RespuestaConsultaPazySalvoLaboratorioDTO> objLaboratorio;
    private List<RespuestaConsultaPazySalvoFinancieraDTO> objFinanciera;
    private List<RespuestaConsultaPazySalvoDeportesDTO> objDeportes;
    private String mensaje;
}

File: app-estudiante/src/app/modelo/RespuestaConsultaPazySalvoDTO.ts
import { RespuestaConsultaPazySalvoLaboratorioDTO } from './RespuestaConsultaPazySalvoLaboratorioDTO';
import { RespuestaConsultaPazySalvoFinancieraDTO } from './RespuestaConsultaPazySalvoFinancieraDTO';
import { RespuestaConsultaPazySalvoDeportesDTO } from './RespuestaConsultaPazySalvoDeportesDTO';

export class RespuestaConsultaPazySalvoDTO {
  codigoEstudiante!: number;
  objLaboratorio!: RespuestaConsultaPazySalvoLaboratorioDTO[];
  objFinanciera!: RespuestaConsultaPazySalvoFinancieraDTO[];
  objDeportes!: RespuestaConsultaPazySalvoDeportesDTO[];
  mensaje!: string;
}